src/screens/login/components/ClouderyViewSwitch.tsx

```diff
@@ -226,7 +226,7 @@ const ClouderyWebView = forwardRef(
         onOpenWindow={openWindowWithInAppBrowser}
         injectedJavaScriptBeforeContentLoaded={run}
         style={{
-          backgroundColor: clouderyTheme?.backgroundColor
+          backgroundColor: clouderyTheme.backgroundColor
```

Comment:
can you remove this change? (or make in a new commit) cc @Ldoppea

Reply:
Done